bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -407,6 +407,78 @@ async def end_nomination_command(self, ctx: Context, user: MemberOrUser, *, reas
         else:
             await ctx.send(f":x: {user.mention} doesn't have an active nomination.")
 
+    @nomination_group.group(name="append", aliases=("amend",), invoke_without_command=True)
+    @has_any_role(*STAFF_ROLES)
+    async def nomination_append_group(self, ctx: Context) -> None:
+        """Commands to append additional text to nominations."""
+        await ctx.send_help(ctx.command)
+
+    @nomination_append_group.command(name="reason")
+    @has_any_role(*STAFF_ROLES)
+    async def append_reason_command(
+        self,
+        ctx: Context,
+        nominee_or_nomination_id: UnambiguousMemberOrUser | int,
+        nominator: UnambiguousMemberOrUser | None = None,
+        *,
+        reason: str
+    ) -> None:
+        """
+        Append additional text to the nomination reason of a specific nominator for a given nomination.
+
+        If nominee_or_nomination_id resolves to a member or user,
+        append the text to the currently active nomination for that person.
+        Otherwise, if it's an int, look up that nomination ID to edit.
+
+        If no nominator is specified, assume the invoker is editing their own nomination reason.
+        Otherwise, edit the reason from that specific nominator.
+
+        Raise a permission error if a non-mod staff member invokes this command on a
+        specific nomination ID, or with an nominator other than themselves.
+        """
+        # If not specified, assume the invoker is editing their own nomination reason.
+        nominator = nominator or ctx.author
+
+        if not any(role.id in MODERATION_ROLES for role in ctx.author.roles):
+            if ctx.channel.id != Channels.nominations:
+                await ctx.send(f":x: Nomination edits must be run in the <#{Channels.nominations}> channel.")
+                return
+
+            if nominator != ctx.author or isinstance(nominee_or_nomination_id, int):
+                raise BadArgument(
+                    "Only moderators can edit specific nomination IDs, "
+                    "or the reason of a nominator other than themselves."
+                )
+
+        if isinstance(nominee_or_nomination_id, int):
+            nomination_id = nominee_or_nomination_id
+            original_nomination = await self.api.get_nomination(nomination_id=nomination_id)
+            if original_nomination:
+                nomination_entries = original_nomination.entries
+            else:
+                await ctx.send(f":x: {nominee_or_nomination_id} is not a valid nomination id.")
+                return
```

Comment:
I think `get_nomination` will raise an error if the id doesn't exist instead of returning `None`.